src/core/task/Task.ts

```diff
@@ -3696,19 +3698,14 @@ export class Task extends EventEmitter<TaskEvents> implements TaskLike {
 
 				await this.say(
 					"api_req_retry_delayed",
-					`${headerText}Retry attempt ${retryAttempt + 1}\nRetrying in ${i} seconds...`,
+					`${headerText}\nAttempt #${retryAttempt + 2} in ${i} seconds...`,
```

Comment:
Not a blocker, but I wonder if we should make retryAttempt start at 1 instead of 0 or something... the + 2 is a bit of a code smell